tools/setup/postgresql-init-dev-db

```diff
@@ -106,6 +106,9 @@ psql -v ON_ERROR_STOP=1 -e -h localhost "$DBNAME_BASE" "$USERNAME" <<EOF
 CREATE SCHEMA zulip;
 EOF
 
+sudo sed -i "s/#shared_preload_libraries = ''/shared_preload_libraries = 'pg_search'/" /etc/postgresql/14/main/postgresql.conf
+sudo systemctl restart postgresql@14-main
```

Comment:
After the package is installed, `pg_search` needs to be added to preload libraries (https://docs.paradedb.com/deploy/pg_search#update-postgresql-conf). I'm not sure what's a good way to do this in dev?

Reply:
Probably dropping a line into `/etc/postgresql/{pgversion}/main/conf.d/zulip-dev.conf` is the way to go here.